src/components/task-plan/homework/exercises.cjsx

```diff
@@ -184,8 +185,10 @@ ExerciseTable = React.createClass
 
   renderExerciseRow: (exerciseId, index, hasTeks) ->
     {section, lo, tagString} = ExerciseStore.getTagStrings(exerciseId)
-    content = ExerciseStore.getContent(exerciseId)
-
+    content = $("<span>#{ExerciseStore.getContent(exerciseId)}</span>")
```

Comment:
I didn't want to wrap it in that component since this is just a quick summary of the problem and should only be on one screen, but I can give it a shot...